content/pages/install/index.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: install and update rulesets
+permalink: /install/
+img: None
+summary: "Instruction to install and update rulesets to osu!lazer."
+---
+
+# download rulesets file
+
+Go to ruleset's repositories release page or go to [rulesets status page](https://rulesets.info/pages/status) and press download button to go to ruleset's repositories release page.
+
+![status page](img/status-download.png)
+
+Then, download a ruleset file by press on `.dll` file in release list.
+
+![sentakki release download](img/sentakki-release-download.png)
+
+# import rulesets to osu!lazer
+
+### step 1 : navigate to osu! data directory
+
+Open the setting menu in osu!, in `General` part there would be a button to `Open osu! folder`. On windows you could navigate to the data folder directly via Windows Explorer, by default this location is `%AppData%/osu!/rulesets`. On MacOS default this location is in application itself.
+
+![open-osu-folder.png](img/open-osu-folder.png)
+
+### step 2: installing the ruleset
+
+Copy the ruleset file (.dll file that you just download) into the rulesets directory, do make sure that duplicate copies of the ruleset is overwritten.
+
+Other rulesets also could share the same folder.
+
+### stpe 3: have fun!
+
+After completing the above steps. If you already open osu!lazer during install you may need to restart osu!lazer. Once osu!lazer is started, you should see the ruleset alongside the standard rulesets.
```

Comment:
After completing the above steps, you may need to restart osu!lazer if it is already open. Once restarted, you should see the ruleset alongside the standard rulesets.